### DISCUSSION TITLE
Microdonaciones a contribuidores

### DISCUSSION THREAD

**Discussion by zebnat** [zebnat/morpho-dimension#23] [Ideas]
# Microdonaciones a contribuidores 🤩

PROPUESTA. Se me ha ocurrido una idea para implementar en morpho dimension.

Los usuarios que contribuyan de forma habitual o hayan contribuído en gran medida a morphoD, serán marcados como `contribuidor` en su cuenta de macrojuegos. Esto (además de que se mostrará en el perfil de macrojuegos también) servirá para que en morphoD tengamos una sección de contribuidores.

## Sección de contribuidores

¿Para qué sirve? Desde aquí podrás hacer MICRODONACIONES. Será algo similar al enviar dinero de macrojuegos, pero sin necesidad de confirmación de correo.

SOLAMENTE se podrá donar a usuarios `contribuidor`.

El sistema te permitirá donarle `macromonedas` `morphocoins`, incluso cantidades pequeñitas como 1000. De esta forma podrás agradecerles que contribuyan de forma gratuíta al proyecto.

Pondremos también que los contribuidores puedan poner links de plataforma de donación como pueden ser `paypalme` o `buymeacoffee`. De este modo quien quiera les podrá donar también con dinero real.

## Contribuir a MorphoD

Actualmente el único contribuidor es `zebnat` pero hablaré con ilustradores de la comunidad por si quieren hacer dibujos que podamos poner en la interfaz de la aplicación. En el futuro si funciona bien, podríamos incluso hablar con Panaworld para que estos ilustradores gráficos, usuarios de la comunidad, puedan también crear algún avatar.

Para salir como contribuidor hay que hacer cosas que tengan bastante valor.

**Comment by TheMiler** [comment]
Buenas Zeb y comunidad. Me gusta mucho esta idea que tienes y estoy a favor de que se haga. Con esto harás que los usuarios quieran apoyar con más entusiasmo a MorphoD. 
¡Un saludo cordial!
TheMiler

**Reply by xxxDivinityxxx** [reply]
Hola Zebnat me parece muy Interesante quisiera saber como Podría tener un Morpho Dimensión en mi perfil de macrojuegos?
Saludos <3 en macrojuegos el nombre es igual a este.

**Comment by Manue1Mart1nez26** [comment]
Se escucha interesante como un puede contribuir en morpho dimension o bueno que es exactamente, he visto actualmente en macrojuegos (acceso a la morpho dimension, ya que hace mucho que no usaba macrojuegos) es una app o una aplicación
